sqlobject/src/test/java/org/jdbi/v3/docs/TestInClauseExpansion.java

```diff
@@ -63,18 +65,21 @@ public void testInClauseExpansion() throws Exception
         ImmutableSet<String> findIdsForNames(@BindIn("names") List<Integer> names);
     }
 
-    public static class ImmutableSetCollectorFactory<T> implements CollectorFactory<T, ImmutableSet<T>> {
+    public static class ImmutableSetCollectorFactory<T> implements CollectorFactory {
```

Comment:
Investigate whether this duplicate implementation of immutable set collector factory is necessary for this test. Remove if redundant.